src/components/HtmlFromAdmin/index.js

```diff
@@ -5,7 +5,7 @@ import SectionHeader from 'components/SectionHeader';
 
 const HtmlFromAdmin = ({ title, content }) => (
   <div className="coa-HtmlFromAdmin">
-    {title && <SectionHeader>{title}</SectionHeader>}
+    {/* {title && <SectionHeader>{title}</SectionHeader>} */}
```

Comment:
I would like an explanation for why code is commented out, and/or just remove it if we don't have plans to put this back into the code.

Reply:
Removed